gazebo_plugins/src/gazebo_ros_tricycle_drive.cpp

```diff
@@ -227,17 +227,15 @@ void GazeboRosTricycleDrive::motorController ( double target_speed, double targe
 {
     double applied_speed = target_speed;
     double applied_angle = target_angle;
-    double applied_steering_speed = 0;
 
     double current_speed = joint_wheel_actuated_->GetVelocity ( 0 );
     if ( wheel_acceleration_ > 0 ) {
         double diff_speed = current_speed - target_speed;
         if ( fabs ( diff_speed ) < wheel_speed_tolerance_ ) {
-            applied_speed = target_speed;
-        } else if ( diff_speed < target_speed ) {
-            applied_speed = current_speed + wheel_acceleration_ * dt;
-        } else {
-            applied_speed = current_speed - wheel_deceleration_ * dt;
+            applied_speed = current_speed;
+        } else if ( fabs(diff_speed) > wheel_acceleration_ * dt ) {
+            if(diff_speed > 0){ applied_speed = current_speed - wheel_acceleration_ * dt;}
+            else              { applied_speed = current_speed + wheel_deceleration_ * dt;}
```

Comment:
I was just reminding you of a comment you made. I don't think we should make huge style changes until we declare what the style should be for these packages